doc/user_guide/index.rst

```diff
@@ -38,32 +33,85 @@
    -----------------------------------------------------------------------------
    Written by R. W. Ford and A. R. Porter, STFC Daresbury Lab
 
+.. only:: html
+
+    .. image:: psyclone_v1.0.png
+        :width: 75%
+        :align: center
+        :alt: PSyclone
+
+PSyclone User's Guide
+=====================
+
+
+.. only:: html
+
+    PSyclone is a code generation system developed to produce domain-specific
+    languages (DSLs) for finite element, finite volume and finite difference
+    codes. Notably, it is used in the
+    `LFRic Project <https://www.metoffice.gov.uk/research/modelling-systems/lfric/>`_,
+    but it is also being extended to support the
+    `GOcean <https://puma.nerc.ac.uk/trac/GOcean>`_ and
```

Comment:
Picky again but GOcean isn't an ocean model, maybe it's a DSL?
How about:
"and it also supports the GOcean (2D, finite difference) DSL. In addition it is being extended to support the processing of existing finite difference codes such as the NEMO ocean model."

Reply:
Done